homeassistant/components/asuswrt/config_flow.py

```diff
@@ -134,6 +137,9 @@ async def async_step_user(self, user_input=None):
 
         errors = {}
         self._host = user_input[CONF_HOST]
+        if user_input.get(CONF_PORT, -1) == 0:
```

Comment:
You don't use the -1 and it's never equal to 0, so you can omit it as the default is `None`.
```suggestion
        if user_input.get(CONF_PORT) == 0:
```

Reply:
Was just to not compare `None` with number, but this check now is useless having the port field optional.